### PR TITLE
Revised messages.it.yml file

### PR DESCRIPTION
**Pull request by matteocoder**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | yes
| Fixed tickets | #...
| License       | MIT

Here is the fully translated messages.it.yml file.

On the other hand, I have other translations to submit that are from already merged pull requests. Does it mean that I need to open new PR for each of them?

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.it.yml [hunk 1]
```diff
@@ -295,43 +297,43 @@ howto:
...
-    # more: 'More…'
+    more: 'Più…'
     intro:
         title: 'Benvenuto su wallabag!'
         paragraph_1: "Un tour in cui ti guideremo per scoprire e che ti mostrerà delle funzionalità che potrebbero interessarti."
         paragraph_2: 'Seguici!'
     configure:
         title: "Configura l'applicazione"
-        # description: 'In order to have an application which suits you, have a look into the configuration of wallabag.'
+        description: 'Per avere un'applicazione che ti soddisfi, dai un'occhiata alla configurazione di wallabag.'
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.it.yml]: Use double quotes here, otherwise it breaks everything.

Comment (matteocoder) [inline src/Wallabag/CoreBundle/Resources/translations/messages.it.yml]: You're right!

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.it.yml]: It seems now we put double quotes everywhere. Plz confirm @wallabag/core

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.it.yml]: I don't know who did that but we should put double only when it's required I think

**Comment by j0k3r** [comment]
Thanks @matteocoder !

**Pull request merged**